README.org

```diff
@@ -883,9 +882,19 @@ In contrast, if you think a package is recommended for every Emacser (especially
 If a package is available in the latest version of Emacs, please remember to add a =[built-in]= tag in the front of the description.
 
 ** Generate "Table of Contents"
-After you have added a package or edited the =README.org=, and before committing and issuing a pull request on this awesome list, use =gen-toc.el= to update the Table of Contents.
 
-Open =gen-toc.el= and =M-x eval-buffer=. Then change buffers to =README.org=. Subsequently, issue =M-x awesome-emacs-gen-toc= in the =README.org= buffer. If a new category of packages has been added in your commit, the Table of Contents section will be revised and updated.
+It is recommended to install [[https://melpa.org/#/toc-org][toc-org]] that will update the Table Of Contents for
```

Comment:
@kuanyui , what are your thoughts on removing gen-toc.el and requiring an external package from melpa (toc-org) to be installed?

toc-org from melpa would be run based on the travis yml file. But is it conceptually better to generate the table of contents based on a present file (what's happening now) or requiring that a packaged be resourced externally (from melpa or github)?

Reply:
Yeah, I also wasn't sure if this is acceptable to you or not. Though, after thinking about it, I decided that both ways are kind of awkward and have their cons. However,  `toc-org` is a more or less popular package (almost 200,000 downloads on MELPA) and it gives you a nice navigation via `C-c C-o` and, technically, it's almost the same as having a `gen-toc.el` file, so I figured this is a reasonable change. Would be interesting to see what you think.

Reply:
I don't sure if require an external package is better... XD

But gen-toc looks a quite popular package, so I think it's ok. Anyway, the only thing we have to ensure for this decision is the generated TOC must be compatible with GitHub's title link.

Reply:
Well, having the links compatible with Github is the main purpose of toc-org =)